tests/test_runner.py

```diff
@@ -1,4 +1,13 @@
 from pytest import Pytester
+from unittest.mock import Mock
+import sys
+
+# Mock config module so that no connections are made at import time leading to exceptions
+sys.modules["config"] = Mock()
+sys.modules["gspread"] = Mock()
+
+from config import config
```

Comment:
```suggestion
from config import config  # noqa: E402
```
since this is a testing and required bypass some linting rules
use the suggestion above, it should solve the issue